### DISCUSSION TITLE
[URGENT]: Too many sleep connections

### DISCUSSION THREAD

**Discussion by Poseidonas** [laravel/framework#36433] [General]
Hello guys,

We have an issue with our database (MariaDB) where fills woth thousands of sleep connections and we cant determine the problem. Any help here?

Thanks!

**Comment by pgrimaud** [comment]
Did you develop something special with queue worker or cron job? You may have an error somewhere which kill a process before disconnecting from DB.

Dirty/temporary workaround : 

I guess you didn't change the MariaDB settings [`wait_timeout`](https://mariadb.com/docs/reference/es/system-variables/wait_timeout/) & [`interactive_timeout`](https://mariadb.com/docs/reference/es/system-variables/interactive_timeout/)? 

By default they're set to 28800 (8 hours).
You can reduce it to 60 or 180 until you find the real problem.

**Reply by Poseidonas** [reply]
Thanks for your attention @pgrimaud :)

we have added some crons, like: Order confirmation (via email), sending mail with tracking id, an another mail when the order changes status. 

Our hosting provider changes the values of wait_timeout & interactive_timeout to 120
